### PR TITLE
Service Mesh doc updates for TP2 release

### PR DESCRIPTION
**Pull request by JStickler**
Updating the service mesh documentation for TP2 which will include:
- Updating the installation guide to include Kiali components and Kiali tutorial
- Updating the Release Notes

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@JStickler, when do you need to release this?

File: servicemesh-install/topics/install.adoc [hunk 1]
```diff
@@ -1,21 +1,227 @@
...
+
+ If deployment_type=openshift then `0.7.2`.
+|===
+
+.Launcher parameters
+|===
+|Component |Parameter |Description |Default
+
+|openshift
+|`use`
```
Comment (brian-avery) [inline servicemesh-install/topics/install.adoc]: user

File: servicemesh-install/topics/install.adoc [hunk 2]
```diff
@@ -1,21 +1,227 @@
...
 
-You can find the https://github.com/Maistra/openshift-ansible/tree/maistra-0.1.0-ocp-3.1.0-istio-1.0.0/istio[operator templates on GitHub].
+The following steps will install the {ProductShortName} operator into an existing {product-title} 3.10 installation; they can be executed from any host with access to the cluster.  Please ensure you are logged in as a cluster admin before executing the following commands.
 
 ```
 $ oc new-project istio-operator
 $ oc new-app -f istio_product_operator_template.yaml --param=OPENSHIFT_ISTIO_MASTER_PUBLIC_URL=<master public url>
 ```
+[NOTE]
+===
```
Comment (brian-avery) [inline servicemesh-install/topics/install.adoc]: These equal signs aren't being formatted properly.

File: servicemesh-install/topics/install.adoc [hunk 3]
```diff
@@ -1,21 +1,227 @@
...
+The OpenShift Master Public URL should be configured to match the public URL of your OpenShift Console, this parameter is required by the Fabric8 Launcher.
+===
+
+
 [[verifying-operator-installation]]
 == Verifying operator installation
```
Comment (brian-avery) [inline servicemesh-install/topics/install.adoc]: Can't comment on it since it's not part of this PR, but see line 256. "To deploy the control plan,". Plan should be plane.

Comment (brian-avery) [reply servicemesh-install/topics/install.adoc]: Also, lines 245-250. The only useful part of this is line 250. We're trimming that down in community docs. May be useful to do here as well.

File: servicemesh-install/topics/tutorial-prometheus.adoc [hunk 4]
```diff
@@ -4,6 +4,7 @@
 TODO
 Need a short overview of Prometheus.
 ////
+This tutorial uses {ProductShortName} and the `bookinfo` tutorial to demonstrate how you can query for metrics using Prometheus.
 
 Prerequisites:
```
Comment (brian-avery) [inline servicemesh-install/topics/tutorial-prometheus.adoc]: Line 55 -- you have odd line breaks here (after name}') that means the commands don't copy/paste correctly.

oc get prometheus -n istio-system -o jsonpath='{.items[*].spec.metrics[*].name}'
requests_total request_duration_seconds request_bytes response_bytes tcp_sent_bytes_total tcp_received_bytes_total[root@slareceived_bytes_totalrequests_total request_duration_seconds request_bytes response_bytes tcp_sent_bytes_total tcp_ 
bash: requests_total: command not found...

File: servicemesh-install/topics/tutorial-kiali.adoc [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+kiali                  kiali-istio-system.127.0.0.1.nip.io                            kiali                  <all>                           None
+prometheus             prometheus-istio-system.127.0.0.1.nip.io                       prometheus             http-prometheus                 None
+tracing                tracing-istio-system.127.0.0.1.nip.io                          tracing                tracing           edge          None
+```
++
+. Launch a browser and navigate to navigate to https://${KIALI_URL} (in the output above, this is `kiali-istio-system.127.0.0.1.nip.io`).  You should see the Kiali console login screen.
++
+image:kiali-login.png[Login Page]
++
+The default username and password are "admin/admin".
```
Comment (brian-avery) [inline servicemesh-install/topics/tutorial-kiali.adoc]: There isn't a default Kiali username and password in OpenShift Service Mesh

File: servicemesh-install/topics/install.adoc [hunk 6]
```diff
@@ -1,21 +1,227 @@
...
+    prefix: openshift-istio-tech-preview/
+    version: 0.2.0
+  jaeger:
+    prefix: distributed-tracing-tech-preview/
+    version: 1.7.0
+    elasticsearch_memory: 1Gi
+  kiali:
+    username: username
+    password: password
+    prefix: kiali/
```
Comment (brian-avery) [inline servicemesh-install/topics/install.adoc]: This is the community prefix. You want the product prefix here (openshift-istio-tech-preview/)

File: servicemesh-install/topics/tutorial-prometheus.adoc (hunk 4 above)
Comment (JStickler) [reply servicemesh-install/topics/tutorial-prometheus.adoc]: Good catch, thanks!

File: servicemesh-install/topics/tutorial-kiali.adoc [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+istio-ingress          istio-ingress-istio-system.127.0.0.1.nip.io                    istio-ingress          http                            None
+istio-ingressgateway   istio-ingressgateway-istio-system.127.0.0.1.nip.io             istio-ingressgateway   http                            None
+jaeger-query           jaeger-query-istio-system.127.0.0.1.nip.io                     jaeger-query           jaeger-query      edge          None
+kiali                  kiali-istio-system.127.0.0.1.nip.io                            kiali                  <all>                           None
+prometheus             prometheus-istio-system.127.0.0.1.nip.io                       prometheus             http-prometheus                 None
+tracing                tracing-istio-system.127.0.0.1.nip.io                          tracing                tracing           edge          None
+```
++
+. Launch a browser and navigate to navigate to https://${KIALI_URL} (in the output above, this is `kiali-istio-system.127.0.0.1.nip.io`).  You should see the Kiali console login screen.
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-kiali.adoc]: double navigate to
I'd remove "(in the output above, this is `kiali-istio-system.127.0.0.1.nip.io`)" and use the annotations to tag the URL in the previous output.

File: servicemesh-install/topics/tutorial-kiali.adoc [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
++
+image:kiali-login.png[Login Page]
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-kiali.adoc]: Why do we need this image?

File: servicemesh-install/topics/tutorial-kiali.adoc [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
++
+
+
+== Graph page
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-kiali.adoc]: Add an anchor.

File: servicemesh-install/topics/tutorial-kiali.adoc [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+
+After you log in you should see the Graph page.  This page shows a graph with all the microservices, connected by the requests going through then. On this page, you can see how the services interact with each other.
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-kiali.adoc]: no should
\*Graph\*
s/then/them

File: servicemesh-install/topics/tutorial-kiali.adoc [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+
+. From the Namespace menu, select bookinfo.  Now the graph only displays the services in the bookinfo application.
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-kiali.adoc]: \*Namespace\*
s/only displays/displays only
Check caps of bookinfo in all of these.

File: servicemesh-install/topics/tutorial-kiali.adoc [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
++
+image:kiali-graph.png[kiali graph]
++
+. Hover over the productpage node.  Note how the graph highlights only the incoming and outgoing traffic from the node.
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-kiali.adoc]: \`productpage\`

File: servicemesh-install/topics/tutorial-kiali.adoc [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+. Click the productpage node.  Note how the details on the right side of the page change to show the productpage details.
+
+== Services page
+
+. Click the Services link in the left navigation. On the Services page you can view a listing of all the services that are running in the cluster, and additional information about them, such as health status and request error rate.
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-kiali.adoc]: s/On the Services page/On the Services page,
s/the cluster,/the cluster

File: servicemesh-install/topics/tutorial-kiali.adoc [hunk 14]
```diff
@@ -0,0 +1,84 @@
...
+. Click the Reviews service to view its details.  Note that there are three different versions of this service.
++
+image:kiali-services-details.png[service health]
++
+. Click the name of one of the services to view additional details about that service.
+
+
+== Istio config page
+
+. Click the Istio Config link in the left navigation. On this page, you can see all of the currently running configurations, such as Circuit Breakers, Destination Rules, Fault Injection, Gateways, Routes, Route Rules, Virtual Services, and so on.
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-kiali.adoc]: no "and so on"

File: servicemesh-install/topics/tutorial-kiali.adoc [hunk 15]
```diff
@@ -0,0 +1,84 @@
...
+image:kiali-istio-config.png[service health]
++
+. Click one of the configurations to view additional information.
++
+image:kiali-istio-config2.png[service health]
+
+
+== Distributed tracing page
+
+Click the Distributed Tracing link in the left navigation. On this page you can see tracing data as provided by Jaeger.  We have previously explored Jaeger in the Jaeger tutorial.
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-kiali.adoc]: No "we"

File: servicemesh-install/topics/tutorial-prometheus.adoc [hunk 16]
```diff
@@ -20,7 +21,7 @@ Prerequisites:
 $ oc get svc prometheus -n istio-system
 ```
 +
-You should see something like the following:
+You will see something like the following:
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-prometheus.adoc]: I'd remove this line and just show the output. :man_shrugging:

File: servicemesh-install/topics/tutorial-prometheus.adoc [hunk 17]
```diff
@@ -33,31 +34,30 @@ prometheus   10.59.241.54   <none>        9090/TCP   2m
 $ curl -o /dev/null http://$GATEWAY_URL/productpage
 ```
 +
-. A route to access the Prometheus user interface should already exist. Query OpenShift for details of the route:
+. A route to access the Prometheus user interface already exists. Query OpenShift for details of the route:
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-prometheus.adoc]: no "OpenShift"

File: servicemesh-install/topics/tutorial-prometheus.adoc [hunk 18]
```diff
@@ -33,31 +34,30 @@ prometheus   10.59.241.54   <none>        9090/TCP   2m
...
 +
 ```
   $ export PROMETHEUS_URL=$(oc get route -n istio-system prometheus -o jsonpath='{.spec.host}')
 ```
 +
-. Launch a browser and navigate to  `http://${PROMETHEUS_URL}`, you should see the Prometheus home screen, similar to this:
+. Launch a browser and navigate to  `http://${PROMETHEUS_URL}`, you will see the Prometheus home screen, similar to the following figure:
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-prometheus.adoc]: comma splice

File: servicemesh-install/topics/tutorial-prometheus.adoc [hunk 19]
```diff
@@ -33,31 +34,30 @@ prometheus   10.59.241.54   <none>        9090/TCP   2m
...
 +
 image::prometheus-home-screen.png[]
 +
-. In the "Expression" field enter `istio_request_duration_seconds_count`, and click the `Execute` button. You should see a screen similar to this:
+. In the *Expression* field enter `istio_request_duration_seconds_count`, and click the `Execute` button. You will see a screen similar to the following figure:
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-prometheus.adoc]: s/field/field,

File: servicemesh-install/topics/tutorial-prometheus.adoc [hunk 20]
```diff
@@ -33,31 +34,30 @@ prometheus   10.59.241.54   <none>        9090/TCP   2m
...
 +
 image::prometheus-metrics.png[]
 +
-. You can narrow down queries by using selectors. For example `istio_request_duration_seconds_count{destination_workload="reviews-v2"}` will only show counters with matching "destination_workload" label. For more information about using queries, see the link:https://prometheus.io/docs/prometheus/latest/querying/basics/#instant-vector-selectors[Prometheus documentation].
+. You can narrow down queries by using selectors. For example `istio_request_duration_seconds_count{destination_workload="reviews-v2"}`  only shows counters with the matching *destination_workload* label. For more information about using queries, see the link:https://prometheus.io/docs/prometheus/latest/querying/basics/#instant-vector-selectors[Prometheus documentation].
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-prometheus.adoc]: s/only shows/shows only

File: servicemesh-install/topics/tutorial-prometheus.adoc [hunk 21]
```diff
@@ -33,31 +34,30 @@ prometheus   10.59.241.54   <none>        9090/TCP   2m
...
 +
-. To list all available Prometheus metrics, use the following command:
+. To list all available Prometheus metrics, execute the following command:
```
Comment (kalexand-rh) [inline servicemesh-install/topics/tutorial-prometheus.adoc]: s/execute/run

File: servicemesh-release-notes/topics/rn-fixed-issues.adoc [hunk 22]
```diff
@@ -8,10 +8,4 @@ Fix - What did we change to fix the problem?
...
 ////
 
-BZ# https://bugzilla.redhat.com/show_bug.cgi?id=000001[000001]
-
-There was a werewolf in the last release.  You would have only encountered this werewolf if it was a full moon and you were outside.  The werewolf has been shot with a silver bullet and no longer exists.
-
-JIRA# https://issues.jboss.org/browse/MAISTRA-5[000005]
-
-There was an unrecognized vampire in the last release.  As a result, the vampire bit a lot of people and sired multiple vampires.  But after the liberal application of sunlight, garlic, and holy water, all vampires have been removed from the current release and you no longer need to wear turtlenecks when going outside.
+https://github.com/Maistra/openshift-ansible/pull/19/[MASITRA-21] -  The default in the installer of 512Mi was too low for tracing.  Increased default Elasticsearch memory from 512 Mb to 1Gb.
```
Comment (kalexand-rh) [inline servicemesh-release-notes/topics/rn-fixed-issues.adoc]: @vikram-redhat, are the GitHub references in this topic and the next topic ok? 

512 MB and 1 GB

File: servicemesh-release-notes/topics/rn-known-issues.adoc [hunk 23]
```diff
@@ -37,13 +37,42 @@ oc process -f istio_product_operator_template.yaml | oc delete -f -
 
 https://issues.jboss.org/browse/MAISTRA-5[MAISTRA-5] - `openshift-ansible-istio-installer-job` pod tries to start but with errors.
 
+Graph layout - The layout for the Kiali graph may render differently, depending on your application architecture and the data to display (number of graph nodes and their interactions).  Because it is difficult if not impossible to create a single layout that could render nicely for every situation, Kiali offers a choice of several different layouts.  To choose a different layout you can choose a different Layout Schema from the Graph Settings menu.
```
Comment (kalexand-rh) [inline servicemesh-release-notes/topics/rn-known-issues.adoc]: s/may/can
no could
s/different layout/different layout,
Surround the UI elements with *

File: servicemesh-release-notes/topics/rn-known-issues.adoc [hunk 24]
```diff
@@ -37,13 +37,42 @@ oc process -f istio_product_operator_template.yaml | oc delete -f -
...
+
+https://issues.jboss.org/browse/MAISTRA-13[MAISTRA-13] - Installer should determine release version from installed components.  At present the installer queries the yaml file, however if the yaml has been modified the installer is not able to remove an older version.
```
Comment (kalexand-rh) [inline servicemesh-release-notes/topics/rn-known-issues.adoc]: no should
s/At present/At present,
s/has been modified/has been modified,

File: servicemesh-release-notes/topics/rn-known-issues.adoc [hunk 25]
```diff
@@ -37,13 +37,42 @@ oc process -f istio_product_operator_template.yaml | oc delete -f -
...
-https://issues.jboss.org/browse/MAISTRA-21[MAISTRA-21] - Increase elasticsearch memory requirement.
-The current default in the installer is 512Mi however this appears to be too low for tracing.
+*Workaround* - If the installation fails because of a previous, older installation, then use the following workaround:
 
-*Workaround* - Include the following in the custom resource definition file.
 ```
-spec:
-  jaeger:
-    elasticsearch_memory: 1Gi
+oc delete namespace devex istio-operator istio-system bookinfo logging foo bar
```
Comment (kalexand-rh) [inline servicemesh-release-notes/topics/rn-known-issues.adoc]: Put a $ before each command.

File: servicemesh-release-notes/topics/rn-known-issues.adoc [hunk 26]
```diff
@@ -37,13 +37,42 @@ oc process -f istio_product_operator_template.yaml | oc delete -f -
...
+oc get clusterroles  | grep istio | awk '{print $1}' | xargs oc delete clusterroles
+
+oc get clusterroles  | grep kiali | awk '{print $1}' | xargs oc delete clusterroles
+
+oc get clusterrolebindings  | grep istio | awk '{print $1}' | xargs oc delete clusterrolebindings
+
+oc get clusterrolebindings  | grep kiali | awk '{print $1}' | xargs oc delete clusterrolebindings
 ```
+
+https://github.com/kiali/kiali/issues/507[Kiali-507] Kiali does not support Internet Explorer 11.  This is because the underlying frameworks do not support Internet Explorer.  In order to access the Kiali console you should be using the two most recent versions of the Chrome, Edge, Firefox or Safari browser.
```
Comment (kalexand-rh) [inline servicemesh-release-notes/topics/rn-known-issues.adoc]: No should

File: servicemesh-release-notes/topics/rn-known-issues.adoc [hunk 27]
```diff
@@ -37,13 +37,42 @@ oc process -f istio_product_operator_template.yaml | oc delete -f -
...
+
+https://issues.jboss.org/browse/KIALI-1284[KIALI-1284] In Istio, a Workload can be any pod or group of pods, regardless where they originate from. They may come from Kubernetes Deployments, Replica Sets or even as a single "orphan" pod. In Kiali the current assumption is that a Workload comes from a Deployment. This should represent the vast majority of the cases.
+
+https://issues.jboss.org/browse/KIALI-1570[KIALI-1570]
+When a graph is loading in the Kiali console, it will initially say that the graph is empty instead of displaying a message that the graph is loading.
```
Comment (kalexand-rh) [inline servicemesh-release-notes/topics/rn-known-issues.adoc]: Maybe "...a message that the graph is empty is displayed instead of a message that the graph is loading."

File: servicemesh-release-notes/topics/rn-known-issues.adoc [hunk 28]
```diff
@@ -37,13 +37,42 @@ oc process -f istio_product_operator_template.yaml | oc delete -f -
...
+
+https://issues.jboss.org/browse/KIALI-1572[KIALI-1572]
+If looking at the Kiali logs you may encounter the following error: `Failed to determine console version from file [/opt/kiali/console/version.txt]. error=open /opt/kiali/console/version.txt: no such file or directory Kiali: Console version: unknown`
```
Comment (kalexand-rh) [inline servicemesh-release-notes/topics/rn-known-issues.adoc]: Maybe "If you see the "ERROR" error message when you view the Kiali logs, ignore it."

File: servicemesh-release-notes/topics/rn-known-issues.adoc [hunk 29]
```diff
@@ -37,13 +37,42 @@ oc process -f istio_product_operator_template.yaml | oc delete -f -
...
+You can safely ignore this error.
+
+https://issues.jboss.org/browse/KIALI-1609[KIALI-1609]
+When we are dealing with very small values (for example, less than 0.0.1 rps) you may encounter some inconsistencies in the graph. We are working on making changes to have this function better when dealing with very small values.
```
Comment (kalexand-rh) [inline servicemesh-release-notes/topics/rn-known-issues.adoc]: No "we"
s/may/might or can

File: servicemesh-release-notes/topics/rn-new-features.adoc [hunk 30]
```diff
@@ -1,15 +1,19 @@
...
-= New features
+
 ////
 Feature – Describe the new functionality available to the customer.  For enhancements, try to describe as specifically as possible where the customer will see changes.
 Reason – If known, include why has the enhancement been implemented (use case, performance, technology, etc.).   For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.  There may not have been a 'problem' previously, but system behaviour may have changed.
 Result – If changed, describe the current user experience
 ////
+= New features Technology Preview 2
+
+The release adds the  Kiali observability console to  {ProductName}.  Kiali provides a number of graphs that let you view the topography and health of the microservices that make up your service mesh.   You can view predefined dashboards that provide detailed request and response metrics (volume, duration, size, TCP traffic) per inbound and outbound traffic.  You can also browse your service mesh by application, workloads, and services to view the health of each element.
```
Comment (kalexand-rh) [inline servicemesh-release-notes/topics/rn-new-features.adoc]: s/let you /you can use to

File: servicemesh-release-notes/topics/servicemesh-overview.adoc [hunk 31]
```diff
@@ -6,11 +6,11 @@
...
 ====
 
-{ProductName} is a platform that provides behavioral insights and operational control over the service mesh, providing a uniform way to connect, secure, and monitor microservice applications.
+{ProductName} is a platform that provides behavioral insight and operational control over the service mesh, providing a uniform way to connect, secure, and monitor microservice applications.
 
-The term *_service mesh_* is often used to describe the network of microservices that make up applications based on a distributed microservice architecture and the interactions between those microservices.  As a service mesh grows in size and complexity, it can become harder to understand and manage.
+The term *_service mesh_* describes the network of microservices that make up applications in a distributed microservice architecture and the interactions between those microservices.  As a service mesh grows in size and complexity, it can become harder to understand and manage.
```
Comment (kalexand-rh) [inline servicemesh-release-notes/topics/servicemesh-overview.adoc]: no asterisks on "service mesh"

File: servicemesh-release-notes/topics/servicemesh-overview.adoc [hunk 32]
```diff
@@ -6,11 +6,11 @@
...
 
-Based on the  open source https://istio.io/[Istio] project, {ProductName}  layers transparently onto existing distributed applications, without requiring any changes in service code.  You add {ProductName}  support to services by deploying a special sidecar proxy throughout your environment that intercepts all network communication between microservices.   You configure and manage the service mesh using the control plane features.
+Based on the  open source https://istio.io/[Istio] project, {ProductName}  adds a transparent layer on existing distributed applications; without requiring any changes to the service code.  You add {ProductName}  support to services by deploying a special sidecar proxy throughout your environment that intercepts all network communication between microservices.   You configure and manage the service mesh using the control plane features.
```
Comment (kalexand-rh) [inline servicemesh-release-notes/topics/servicemesh-overview.adoc]: no ;

**Review by kalexand-rh — COMMENTED** [review]
@JStickler, PTAL

File: servicemesh-install/topics/install-overview.adoc [hunk 33]
```diff
@@ -4,30 +4,39 @@
...
-* istio-system project (16 pods)
+* istio-system project (17 pods)
 
-You first create the `istio-operator` project and create a Kubernetes operator.  This operator defines and monitors a custom resource that manages the deployment, updating, and deletion of the Service Mesh components.
+You first create a Kubernetes _operator_. This operator defines and monitors a _custom resource_ that manages the deployment, updating, and deletion of the Service Mesh components.
 
-The operator creates an Ansible job that runs an Ansible playbook that performs the following installation and configuration tasks automatically:
+Depending on how you define the custom resource file, you can install one or more of the following components when you install the {ProductShortName}:
+
+* *Istio* - based on the  open source https://istio.io/[Istio] project, lets you connect, secure, control, and observe the microservices that make up your applications.
```
[the remark this answers is not in the document]
Comment (JStickler) [reply servicemesh-install/topics/install-overview.adoc]: DL?

File: servicemesh-install/topics/servicemesh-architecture.adoc [hunk 34]
```diff
@@ -3,13 +3,13 @@
...
+* The *data plane* is composed of a set of intelligent proxies deployed as sidecars. These proxies intercept and control all inbound and outbound network communication between microservices in the service mesh; sidecar proxies also communicate with Mixer, the general-purpose policy and telemetry hub.
 
 * The *control plane* is responsible for managing and configuring proxies to route traffic, and configuring Mixers to enforce policies and collect telemetry.
 
-The components that make up the data plane and the control plane are as follows:
+The components that make up the data plane and the control plane are:
 
-* *Envoy proxy* is the _data plane_ component which  intercepts all inbound and outbound traffic for all services in the service mesh. Envoy is deployed as a sidecar to the relevant service in the same pod.
+* *Envoy proxy* is the _data plane_ component that intercepts all inbound and outbound traffic for all services in the service mesh. Envoy is deployed as a sidecar to the relevant service in the same pod.
```
[the remark this answers is not in the document]
Comment (JStickler) [reply servicemesh-install/topics/servicemesh-architecture.adoc]: DL?

**Comment by vikram-redhat** [comment]
Hey @JStickler - confirming that there is no reason that the ServiceMesh docs shouldn't carry on to the 3.11 release branch as well?

File: servicemesh-install/topics/install-overview.adoc (hunk 33 above)
[the remark this answers is not in the document]
Comment (kalexand-rh) [reply servicemesh-install/topics/install-overview.adoc]: definition list

**Comment by kalexand-rh** [comment]
I think it needs to land on 3.10 and 3.11. @JStickler, does that sound right to you?

File: servicemesh-install/topics/install.adoc [hunk 35]
```diff
@@ -1,33 +1,239 @@
...
+
+|prefix
+|valid image repo
+|Which prefix to apply to the Kiali image names used in docker pull.
+|If deployment_type=origin the default is `kiali/`.
+
+ If deployment_type=openshift the default is `openshift-istio-tech-preview/`.
+
+|version
+|valid Kiali tag
```
[the remark this answers is not in the document]
Comment (JStickler) [reply servicemesh-install/topics/install.adoc]: Yes, they are actually different in the different distros.

**Comment by JStickler** [comment]
@vikram-redhat and @kalexand-rh , yes, this should go into 3.10 and 3.11 branches.
